src/basic/fs-util.c

```diff
@@ -705,15 +705,17 @@ int inotify_add_watch_fd(int fd, int what, uint32_t mask) {
 }
 
 int inotify_add_watch_and_warn(int fd, const char *pathname, uint32_t mask) {
+        int r = -1 ;
```

Comment:
no need to initialize this, we'll overrid in the next line anyway.

also, there's a spurious space before the ;

Reply:
incorporated comments.